### DISCUSSION TITLE
Amethyst's Two Pane layout possible? Just two most recent windows with others hidden.

### DISCUSSION THREAD

**Discussion by simplygood** [asmvik/yabai#1541] [Q&A] [answered]
Is it possible to have Amethyst's Two Pane layout in yabai? It shows only two most recent windows and hides all others automatically. yabai's stacking method may work similarly, but I would have to stack windows manually.

**Comment by Bellavene** [comment, marked as answer]
It is possible to adapt this script
https://github.com/koekeishiya/yabai/issues/1609
